hs_core/views/__init__.py

```diff
@@ -380,7 +380,7 @@ class FilterForm(forms.Form):
     start = forms.IntegerField(required=False)
     published = forms.BooleanField(required=False)
     edit_permission = forms.BooleanField(required=False)
-    creator = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
```

Comment:
The creator field is used in:

- hs_core.views.serializers.ResourceListRequestValidator
- hs_core.views.serializers.ResourceListItemSerializer
- hs_core.views.serializers.ResourceListItem
- hs_core.views.ResourceToListItemMixin
- hs_core.view.resource_rest_api.ResourceList
- hs_core.view.resource_rest_api.SystemMetadataRetrieve

Can you look into refactoring these to be consistent with your change? (i.e. so that searching/filtering via the REST API works like it does in the web client).